File: URI/1 - INICIANTE/C/1113 - CrescenteEDecrescente.c
#include <stdio.h>

int main()
{
	int x = 0, n1, n2;
	
	while(x == 0)
	{
		scanf(" %d %d", &n1, &n2);
		if(n1 != n2)
		{
			if(n1 > n2)
			{
				printf("Decrescente\n");
			}else{
				printf("Crescente\n");
			}
		}else
		{
			x = 1;
			break;
		}
		
	}
	
	return 0;
}

File: URI/1 - INICIANTE/Python/1113 - CrescenteEDecrescente.py
x = 0
while x == 0:
    n1,n2 = input().split(' ')
    n1 = int(n1)
    n2 = int(n2)
    if n1 != n2:
        if n1 > n2:
            print("Decrescente")
        else:
            print("Crescente")
    else:
        x = 1
        break